azure-pipelines.yml

```diff
@@ -0,0 +1,87 @@
+      displayName: Functional Migration Tests
+    - bash: ./dotnet-run.sh dotnet run --project test/EFCore.MySql.IntegrationTests -c Release testMigrate
+      displayName: Integration Tests applying migrations
+    - bash: ./dotnet-run.sh ./test/EFCore.MySql.IntegrationTests/scripts/scaffold.sh
+      displayName: Integration Tests scaffolding
+    - bash: rm -rf ./test/EFCore.MySql.IntegrationTests/Scaffold
+      displayName: Remove scaffolding files
+    - bash: ./dotnet-run.sh dotnet test -c Release test/EFCore.MySql.IntegrationTests
+      displayName: Integration Tests with EF_BATCH_SIZE=1
+    - bash: export EF_BATCH_SIZE="10" && ./dotnet-run.sh dotnet test -c Release test/EFCore.MySql.IntegrationTests
+      displayName: Integration Tests with EF_BATCH_SIZE=10
+    - bash: export EF_RETRY_ON_FAILURE="3" && ./dotnet-run.sh dotnet test -c Release test/EFCore.MySql.IntegrationTests
+      displayName: Integration Tests with EF_RETRY_ON_FAILURE=3
+    - bash: ./dotnet-run.sh ./test/EFCore.MySql.IntegrationTests/scripts/legacy.sh
+      displayName: Integration Tests legacy migrations
+    - bash: export EF_SCHEMA="pomelo_test2" && ./dotnet-run.sh ./test/EFCore.MySql.IntegrationTests/scripts/rebuild.sh
+      displayName: Integration Tests Building migrations with EF_DATABASE=pomelo_test2
+    - bash: export EF_SCHEMA="pomelo_test2" && ./dotnet-run.sh dotnet test -c Release test/EFCore.MySql.IntegrationTests
+      displayName: Integration Tests with EF_SCHEMA=pomelo_test2
+
+  - job: Windows
+    pool:
+      vmImage: 'windows-2019'
+    continueOnError: true
+    steps:
+    - pwsh: |
+        $ErrorActionPreference = "Stop"
+        
+        choco install mysql
+        C:\tools\mysql\current\bin\mysql.exe -h localhost -u root -e "ALTER USER 'root'@'localhost' IDENTIFIED BY 'Password12!';"
+
+        cp test\EFCore.MySql.FunctionalTests\config.json.example test\EFCore.MySql.FunctionalTests\config.json
+        cp test\EFCore.MySql.IntegrationTests\appsettings.ci.json test\EFCore.MySql.IntegrationTests\appsettings.json
+        cp test\EFCore.MySql.IntegrationTests\config.json.example test\EFCore.MySql.IntegrationTests\config.json
+        .\run.ps1 install-tools
+        .\dotnet-run.ps1 .\test\EFCore.MySql.IntegrationTests\scripts\rebuild.ps1
+      displayName: Setup
+    - pwsh: .\build.cmd "/p:SkipTests=true"
+      displayName: Build Project
+    - pwsh: .\dotnet-run.ps1 dotnet test test\EFCore.MySql.Tests
+      displayName: Tests
+    - pwsh: .\dotnet-run.ps1 dotnet test -c Release test\EFCore.MySql.FunctionalTests --filter "FullyQualifiedName~Pomelo.EntityFrameworkCore.MySql.FunctionalTests.MigrationSqlGeneratorMySqlTest|FullyQualifiedName~Pomelo.EntityFrameworkCore.MySql.FunctionalTests.MigrationSqlGeneratorMySql55Test"
```

Comment:
What is the reason for only testing `MigrationSqlGeneratorMySqlTest` and `MigrationSqlGeneratorMySql55Test` of the `FunctionalTests` suite?

Reply:
Historically TravisCI couldn't handle the full test suite (it would freeze) so I had to just pick some to run as part of the CI suite.

I'd like to move to the full test suite in CI if it's feasible. That probably means ignoring all of the failing tests though